Model/Rate/Result.php

```diff
@@ -343,29 +348,29 @@ private function getPrice($settingPath): float
         $settingFee = 0;
 
         // Explode settingPath like: myparcelnl_magento_postnl_settings/delivery/only_recipient/signature
-        $settingPath = explode("/", $settingPath);
+        $settingPath = explode('/', $settingPath);
 
         // Check if the selected delivery options are delivery, only_recipient and signature
         // delivery/only_recipient/signature
-        if (isset($settingPath[2], $settingPath[3]) && 'delivery' === $settingPath[1]) {
-            $settingFee += (float) $this->myParcelHelper->getConfigValue($settingPath[0] . '/' . $settingPath[1] . '/' . $settingPath[2] . '_' . 'fee');
-            $settingFee += (float) $this->myParcelHelper->getConfigValue($settingPath[0] . '/' . $settingPath[1] . '/' . $settingPath[3] . 'fee');
+        if (isset($settingPath[self::THIRD_PART], $settingPath[self::FOURTH_PART]) && 'delivery' === $settingPath[1]) {
```

Comment:
Prop ze dan meteen ff in variabelen en vervang wat van die concatenation door string interpolation of sprintf, dat zou een hoop visuele terreur schelen. Kan je in 3 sec doen met de intentions van phpstorm 💡